runtime/src/comm/ofi/comm-ofi.c

```diff
@@ -6086,20 +6030,14 @@ chpl_comm_nb_handle_t rmaPutFn_msgOrdFence(void* myAddr, void* mrDesc,
     //
     flags = FI_INJECT;
   }
-  if (tcip->bound && bitmapTest(tcip->amoVisBitmap, node)) {
+  if (bitmapTest(tcip->amoVisBitmap, node)) {
```

Comment:
why is `tcip->bound` no longer checked here?

Reply:
It's not needed and kind of confused me when I first saw it. `bitmapTest` will only return `true` if `tcip->bound` is `true`, and `tcip->bound` isn't checked in other similar conditionals so I removed it.